### ISSUE TITLE
Provide OSGI support

### ISSUE THREAD

**Issue by fabiolimace** [f4b6a3/ulid-creator#16]

**Comment by fabiolimace** [comment]
Added OSGi entries to `MANIFEST.MF`:

```
Bundle-ManifestVersion: 2
Bundle-Name: ulid-creator
Bundle-SymbolicName: com.github.f4b6a3.ulid
Bundle-Version: 4.1.0
Export-Package: com.github.f4b6a3.ulid
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

The OSGi symbolic name is the same as the root package name: `com.github.f4b6a3.ulid`

**Comment by fabiolimace** [comment]
Released version 4.1.0
